.github/FUNDING.yml

```
@@ -0,0 +1,10 @@
+github: # Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
+patreon: # Replace with a single Patreon username
+open_collective: # Replace with a single Open Collective username
+ko_fi: # Replace with a single Ko-fi username
+tidelift: # Replace with a single Tidelift platform-name/package-name e.g., npm/babel
+community_bridge: # Replace with a single Community Bridge project-name e.g., cloud-foundry
+liberapay: # Replace with a single Liberapay username
+issuehunt: # Replace with a single IssueHunt username
+otechie: # Replace with a single Otechie username
+custom: https://pypi.org/sponsor # Replace with up to 4 custom sponsorship URLs e.g., ['link1', 'link2']
```

```suggestion
custom: https://pypi.org/sponsor/ # Replace with up to 4 custom sponsorship URLs e.g., ['link1', 'link2']
```
to avoid a redirect :)

---

Uhhhhhh, I can't accept this in the mobile. UI. :(

---

I added quotes, as recommended in GitHub documentation for this stuff. :)